tslearn/clustering.py

```diff
@@ -731,20 +734,27 @@ def _fit_one_init(self, X, rs):
 
         return self
 
-    def fit(self, X, y=None):
+    def fit(self, X, initial_centroids=None, y=None):
         """Compute k-Shape clustering.
 
         Parameters
         ----------
         X : array-like of shape=(n_ts, sz, d)
             Time series dataset.
+        initial_centroids: array-like of shape=(n_clusters, sz, d) of initial centroids
+            Time series dataset.
         """
 
         X_ = to_time_series_dataset(X)
         self._norms = numpy.linalg.norm(X_, axis=(1, 2))
-        X_ = TimeSeriesScalerMeanVariance(mu=0., std=1.).fit_transform(X_)
```

Comment:
Why is this line removed?

Reply:
Seems like the code has changed to 

`self._norms = numpy.linalg.norm(X_, axis=(1, 2))` and `self._norms_centroids = numpy.linalg.norm(self.cluster_centers_, axis=(1, 2))`. I would also use the Normalizers from tslearn instead of using the numpy norm function...